### ISSUE TITLE
Data catalog page content

### ISSUE THREAD

**Issue by huard** [DACCS-Climate/marble-website#32]
I suggest that at some point in the future, the data catalog page actually holds a federated catalog.

**Comment by mishaschwartz** [comment]
Some thoughts about how to do this:

- include a directive to build the stac-browser static site when running `build.py` (similar to how we're building tutorials now)
- customize the landing page to show a list of catalogs (from each node in the network instead of from https://stacindex.org/) 

This way we don't actually need to have a separate catalog for the website, we just need to point the static stac-browser at the appropriate URLs

**Comment by dchandan** [comment]
I think it would be good to point the browser to a single STAC catalog, say from the `UofT` node, that provides a global view of the data on the network rather than have a list of catalogs from each of the nodes. A global view would underpin one of the key objective of the project, which is to allow users to be able to use data from any node in the network. This would also help a user search the entire network for what they are looking for from a single location, rather than have to search in the catalogs of each of the nodes one-by-one.

**Comment by mishaschwartz** [comment]
@dchandan Agreed but we should have some way to provide an alternate catalog if UofT is down for some reason (I don't want to website to rely entirely on one specific node being operational).

**Comment by dchandan** [comment]
Good point.
